main.sh

```diff
@@ -44,6 +44,16 @@ if [ -n "${GITHUB_OUTPUT+x}" ]; then
   echo "project=$ProjectName" >>"$GITHUB_OUTPUT"
   echo "axe_url=$SERVER_URL$AxeURL" >>"$GITHUB_OUTPUT"
   echo "issue_count=$IssueCount" >>"$GITHUB_OUTPUT"
+  echo "issues_over_a11y_threshold=$IssuesOverA11yThreshold" >>"$GITHUB_OUTPUT"
+fi
+
+if [ "$IssuesOverA11yThreshold" -gt 0 ]; then
+  if [ "$IssueCount" -gt 0 ]; then
```

Comment:
This doesn't seem quite complete; I'd expect the purpose of the feature to be that if the user has enabled a11y_threshold, the script should only be `exit 1`ing if `$IssuesOverA11yThreshold` > 0, even if `$IssueCount` > 0. Am I misunderstanding the feature?

Reply:
Ya so if the user has `enable_a11y_threshold` enabled then do the following: 

`IssuesOverA11yThreshold` - are all violations based on your a11y threshold 
`IssueCount` - are all violations

If `IssuesOverA11yThreshold` is greater than 0, we want to print the count of violations of their a11y threshold _and_ the count of all of the violations that are not based on their a11y threshold. Say you only have critical and serious enabled, the message would be something like: 

```
Found 10 accessibility violations
Found 5 accessibility violations over your a11y threshold
```

So we notify the user that there are still violations outstanding outside of their a11y threshold

Otherwise we'll `exit 0` (good catch I missed that here - updating)